docker-compose.yml

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-x-superset-image: &superset-image uchimera.azurecr.io/cccs/superset:latest
+x-superset-image: &superset-image apache/superset:latest-dev
```

Comment:
Is it what we want here?

Reply:
yes, reverting to default compose file

the image locally will be called apache/superset:latest-dev

but of course our build pipeline will build the uchimera image to deploy on u-dev, u-prod, pb

Reply:
there is no need to change the local image name. This makes us more align with the apache/superset repo, less changes of our own